src/List/Extra.elm

```diff
@@ -548,18 +545,17 @@ stableSortWith pred list =
         List.sortWith predWithIndex listWithIndex |> List.map first
 
 
-{-| Swap two values in a list by index. Returns the updated list if both indices
-are in range, or Nothing if both are out of range. If the same index is
-supplied twice the original list is returned.
+{-| Swap two values in a list by index.
+Returns the updated list if both indices are in range,
+Returns the updated list if both indices are in range, or the original list if an index is out of range.
+If the same index is supplied twice the original list is returned.
 -}
-swapAt : Int -> Int -> List a -> Maybe (List a)
+swapAt : Int -> Int -> List a -> List a
 swapAt index1 index2 l =
-    if index1 == index2 then
-        Just l
-    else if index1 > index2 then
+    if index1 > index2 then
         swapAt index2 index1 l
```

Comment:
Good idea, I was tempted to do that, but I didn't introduced the variable name so I wanted this PR to be focused on one change only.

But I'm doing it with great pleasure now that somebody else is annoyed by this variable name!